### PR TITLE
Feature/test revoke delete

### PR DESCRIPTION
**Pull request by xakraz**

### CODE REVIEW

File: src/puppet_apis/puppetcacli.py [hunk 1]
```diff
@@ -358,3 +358,23 @@ def status(self, hostname):
...
             pass
+
+
+    def revoke(self, hostname):
+        cert_status = self.status(hostname)
+        if 'state' in cert_status and cert_status['state'] == 'signed':
+            self.logger.info("     * Revoking certficate for {}".format(hostname))
+            return bool(self.puppet_ca_client.revoke(hostname))
+
+        self.logger.info("     * Certifiate for node {} is not signed ({})".format(hostname, cert_status['state']))
```
Comment (aladjadj) [inline src/puppet_apis/puppetcacli.py:369]: Typo ? "Certifiate"

File: src/puppet_apis/puppetcacli.py [hunk 2]
```diff
@@ -358,3 +358,23 @@ def status(self, hostname):
...
+        return False
+
+
+    def delete(self, hostname):
+        cert_status = self.status(hostname)
+        if 'state' in cert_status and cert_status['state'] == 'revoked':
+            self.logger.info("     * Deleting certficate for {}".format(hostname))
+            return bool(self.puppet_ca_client.delete(hostname))
+
+        self.logger.info("     * Certifiate for node {} is not revoked yet ({})".format(hostname, cert_status['state']))
```
Comment (aladjadj) [inline src/puppet_apis/puppetcacli.py:379]: Typo ? "Certifiate"

**Comment by aladjadj** [comment]
Wait to PR #1 is merged to run CI

File: src/puppet_apis/puppetcacli.py (hunk 1 above)
Comment (xakraz) [reply src/puppet_apis/puppetcacli.py:369]: Thx I will update it

**Pull request closed without merging**